transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/TransactionOutbox.java

```diff
@@ -65,24 +70,15 @@ static TransactionOutboxBuilder builder() {
   ParameterizedScheduleBuilder with();
 
   /**
-   * Identifies any stale tasks queued using {@link #schedule(Class)} (those which were queued more
-   * than supplied {@link TransactionOutboxBuilder#attemptFrequency(Duration)} ago and have been
-   * tried less than {@link TransactionOutboxBuilder#blockAfterAttempts(int)} )} times) and attempts
-   * to resubmit them.
+   * Identifies any stale tasks queued by checking unprocessed entries sorted by date.
```

Comment:
@rourei we should revisit this comment too, seems to be outdated and the original one could be fitting again.